src/annotator/highlighter.js

```diff
@@ -2,14 +2,6 @@ import { isNodeInRange } from './range-util';
 
 const SVG_NAMESPACE = 'http://www.w3.org/2000/svg';
 
-function isCSSPropertySupported(property, value) {
-  if (typeof CSS !== 'function' || typeof CSS.supports !== 'function') {
```

Comment:
The `typeof CSS !== 'function'` expression is the part that incorrectly failed in Firefox and the newest versions of Chrome.